float32 referenceVelocity
float32 referenceSteerAngle